push.pl

```diff
@@ -81,17 +81,32 @@ sub prompt_for_edit {
 		# TODO https://github.com/docker/hub-beta-feedback/issues/238
 		my $fullUrl = "$githubBase/$proposedFile";
 		my $note = "**Note:** the description for this image is longer than the Hub length limit of $lengthLimit, so has been trimmed.  The full description can be found at [$fullUrl]($fullUrl).  See [docker/hub-beta-feedback#238](https://github.com/docker/hub-beta-feedback/issues/238) for more information.\n\n";
-		$proposedText = $note . substr $proposedText, 0, ($lengthLimit - length($note));
+		
+		# if our text is too long for the Hub length limit, let's first try removing the "Supported tags" list and add our note and see if that's enough to let us put the full image documentation
+		my $trimmedText = $proposedText;
+		$trimmedText =~ s%^# Supported tags and respective `Dockerfile` links\n.*?\n(?=# |\[)%$note%ms;
```

Comment:
Perhaps we just change the note for this version to include an extra blurb explicitly pointing out where to find the now replaced "Supported tags and respective `Dockerfile` links" section?

Reply:
I suppose it could include an even more targeted link directly to the anchor, ala https://github.com/docker-library/docs/blob/master/adoptopenjdk/README.md#supported-tags-and-respective-dockerfile-links :thinking:

Reply:
Updated with a more specific/relevant note for this case (and it now keeps the heading). :+1: